front/app/components/UI/QuillEditor/Toolbar.tsx

```diff
@@ -47,9 +47,10 @@ interface Props {
   limitedTextFormatting?: boolean;
   withCTAButton?: boolean;
   isButtonsMenuVisible: boolean;
-  noImages?: boolean;
-  noVideos?: boolean;
-  noAlign?: boolean;
+  noImages: boolean;
```

Comment:
I almost forgot to pass noLinks in a spot, so I've made these noX props required everywhere, except in the outermost layer (QuillEditor component).